website/www/site/content/en/powered-by/commercial/dataflow.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Cloud Dataflow"
+type: commercial
+categories:
+  - Cloud & Data Analytics
+icon:
```

Comment:
Do we have icons for these?

Reply:
I do not have the assets. In the staged version they render correctly without the icons.